### PR TITLE
replaced old RD command with DELTREE, added new commands RD and REN, …

### PR DESCRIPTION
**Pull request by DavinderJolly**
…and made word_completer and style instance variables

### CODE REVIEW

File: main/main.py [hunk 1]
```diff
@@ -160,4 +194,29 @@ def start_repl(self) -> None:
 
 
 if __name__ == "__main__":
-    Repl().start_repl()
+    word_completer = WordCompleter(
```
Comment (abir0) [inline main/main.py:197]: I think putting `word_completer` and `style` variable inside the `Repl` class would be better as they don't need to be changed from the outside.

File: main/main.py [hunk 2]
```diff
@@ -90,9 +84,39 @@ def remove_dir(self, dir_path: str) -> None:
             if child.is_file():
                 child.unlink()
             else:
-                self.remove_dir(str(child))
+                self.del_tree(str(child))
         path.rmdir()
 
+    def remove_dir(self, dir: str) -> None:
```
Comment (abir0) [inline main/main.py:90]: These seems good

**Review by abir0 — CHANGES_REQUESTED** [review]

File: main/main.py (hunk 2 above)
Comment (DavinderJolly) [reply main/main.py:90]: that would look better initially but i think just leaving that to an argument would make the code a bit more modular if it needs to be imported and customized a bit  outside of the function

File: main/main.py (hunk 1 above)
Comment (DavinderJolly) [reply main/main.py:197]: that would look better initially but i think just leaving that to an argument would make the code a bit more modular if it needs to be imported and customized a bit outside of the function

Comment (abir0) [reply main/main.py:197]: Yeah for styling that's true. But do we need to customize the commands list from outside?

Comment (DavinderJolly) [reply main/main.py:197]: True changing that :)

Comment (DavinderJolly) [reply main/main.py]: added the word completer directly to the class and also added an argument to add extra words

**Review by abir0 — APPROVED** [review]
seems fine

**Pull request merged**